relay-server/src/actors/outcome.rs

```diff
@@ -304,6 +306,9 @@ pub struct TrackRawOutcome {
     /// The source of the outcome (which Relay sent it)
     #[serde(default, skip_serializing_if = "Option::is_none")]
     source: Option<String>,
+    /// The data categories of the events
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub event_category: Option<DataCategory>,
```

Comment:
Sorry, I'm just realizing that this needs to be called `category`. This directly maps to the schema on the Kafka topic, and both Sentry and the consumer already expect `category` for this.